### DISCUSSION TITLE
How to init nuxt auth async

### DISCUSSION THREAD

**Discussion by ozancakir** [nuxt-community/auth-module#1466] [Q&A]
I have a very slow user service on backend, my app loads awaits 'user' request to finish. Sometimes 'user' detail request on my backend fails (and its expected we cannot solve this issue atm.) and the request take 10 seconds to timeout. In meanwhile the whole app is waiting that user request to finish.

Is there a point that I am missing and how can I solve this issue? Thanks for your help 
i am using refresh scheme with   watchLoggedIn: false, redirect: false 
![image](https://user-images.githubusercontent.com/36998030/131797748-628c16fe-8a30-49f1-b29e-4530d73d5c9e.png)

**Comment by Tragio** [comment]
@ozancakir you can try to use the autoFetch to false it can save you a roundtrip but will not solve the situation https://auth.nuxtjs.org/schemes/local#user maybe it's time to start applying some optimizations to the backend, compression and caching to reduce those numbers.

**Comment by ozancakir** [comment]
Thank you for your answer. I have tried autoFetch:false but the issue is not solved. We probably need to came up with another solution since we can't change serverside config.
